From 00208c5ba5fa2c7477272792c5f6fc01d7c86e53 Mon Sep 17 00:00:00 2001
From: Guillaume Aubertin <guillaume.aubertin@intel.com>
Date: Wed, 22 May 2013 16:47:11 +0200
Subject: fixing gps header for SUPL2 wlan reference location

BZ: 118007

Modifying flag names to make it more generic, fixing a typo in
AGPS_REF_LOCATION_TYPE_MAC definition and adding AGPS_REF_LOCATION_END.

Category: AOSP improvement
Domain: CWS_GNSS-HAL
Severity: Medium
Origin: Internal
Upstream-Candidate: yes

Change-Id: If4c0f1c634260862976a3ae45259c4a50f87045b
Orig-Change-Id: Iba13c9d3cea5bd407b5844aec63ca39a03c01d6e
Signed-off-by: Guillaume Aubertin <guillaume.aubertin@intel.com>
Signed-off-by: jerome Pantaloni <jeromex.pantaloni@intel.com>
Reviewed-on: http://android.intel.com:8080/122238
Reviewed-by: buildbot <buildbot@intel.com>
Reviewed-by: Peix, Fabien <fabien.peix@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
---
 include/hardware/gps.h |    7 ++++---
 1 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/include/hardware/gps.h b/include/hardware/gps.h
index 458b5b4..a168e7a 100644
--- a/include/hardware/gps.h
+++ b/include/hardware/gps.h
@@ -192,9 +192,10 @@ typedef uint16_t AGpsStatusValue;
 /** AGPS data connection failed */
 #define GPS_AGPS_DATA_CONN_FAILED   5
 
+#define AGPS_REF_LOCATION_END               0
 #define AGPS_REF_LOCATION_TYPE_GSM_CELLID   1
 #define AGPS_REF_LOCATION_TYPE_UMTS_CELLID  2
-#define AGPS_REG_LOCATION_TYPE_MAC          3
+#define AGPS_REF_LOCATION_TYPE_MAC          3
 
 /** Network types for update_network_state "type" parameter */
 #define AGPS_RIL_NETWORK_TYPE_MOBILE        0
@@ -632,8 +633,8 @@ struct gps_device_t {
 #define AGPS_RIL_REQUEST_SETID_IMSI     (1<<0L)
 #define AGPS_RIL_REQUEST_SETID_MSISDN   (1<<1L)
 
-#define AGPS_RIL_REQUEST_REFLOC_CELLID  (1<<0L)
-#define AGPS_RIL_REQUEST_REFLOC_MAC     (1<<1L)
+#define AGPS_REQUEST_REFLOC_CELLID  (1<<0L)
+#define AGPS_REQUEST_REFLOC_MAC     (1<<1L)
 
 typedef void (*agps_ril_request_set_id)(uint32_t flags);
 typedef void (*agps_ril_request_ref_loc)(uint32_t flags);
-- 
1.7.4.1

